packages/web-app-files/src/fileTypeColorMappings.json

```
@@ -0,0 +1,107 @@
+{
+    "7z": "#FFCF01",
+    "accdb": "var(--oc-color-text-default",
```

```suggestion
    "accdb": "var(--oc-color-text-default)",
```
messed it up big time by omitting the closing brackets here -.- someone please fix for the whole file

---

@pascalwengerter moved it to ODS and yes fixed that one ;)